docs/javascript/v2/guides/deployment-iframe-integration.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Deployment and Iframe Integration
+nav: 2.7
+---
+
```

Comment:
adding an overview section to inform the users of the end result with the outline of steps to be followed will be good.

Reply:
and mentioning the benefits of using iframe is something folks will love :)